profile/Readme.md

```diff
@@ -5,55 +5,26 @@
 
 <p align="center" style="text-align: center;">
   <a href="https://github.com/TKMM-Team/Tkmm/releases">
-    <img src="https://img.shields.io/github/v/tag/TKMM-Team/Tkmm?style=for-the-badge&logoColor=C71B42&color=C71B42&labelColor=2A2C33&logo=github&label=Version" alt="Releases' YouTube"/>
-  </a> &nbsp;
-  <a href="https://discord.com/invite/3tNUBWQRXv">
-    <img src="https://img.shields.io/discord/1179611100183011429?style=for-the-badge&logoColor=3b83c8&color=3b83c8&labelColor=2A2C33&logo=discord&label=discord" alt="Discord"/>
-  </a> &nbsp;
```

Comment:
Why replace the badges? They were already perfect the way they were

Reply:
I think I was trying to match the look of the bottom badges for consistency, so I regenerated the badges, and I might've missed some leading to the removed badges: I can resolve this by making all badges use the same style, and just changing the colour pallate..

Reply:
Replaced the badges with the old ones and moved the second row up!